notebooks/working/investigating_deconfounder.py

```diff
@@ -338,41 +314,54 @@
     ax.axvline(x=obs_ll_per_zi[subject_no], linestyle='--')
 
 plt.show()
-# -
 
-# ## Visualizing the substitute confounder against the true confounder
+# +
+overall_ll_est = rep_ll_per_zi.mean(axis=1)
 
-df.plot(kind='scatter', x = 'confounder_PCA_SKLEARN', y = 'confounder')
+overall_ll_obs = obs_ll_per_zi.mean()
 
-df.plot(kind='scatter', x = 'confounder_PPCA', y = 'confounder')
+sns.kdeplot(overall_ll_est).set_title("Predictive check: Population")
+plt.axvline(x=overall_ll_obs, linestyle='--')
+# -
 
-df.plot(kind='scatter', x = 'confounder_PCA_CV', y = 'confounder')
+overall_ll_est.shape
```

Comment:
I assume this line is here for personal debugging purposes? If not, I'd add a comment or print statement denoting the importance of looking at `overall_ll_est.shape`.